src/mon/OSDMonitor.cc

```diff
@@ -1073,14 +1130,17 @@ void OSDMonitor::prime_pg_temp(
   const OSDMap& next,
   pg_t pgid)
 {
-  if (mon->monmap->get_required_features().contains_all(
+  if (mon->monmap->get_required_features().empty()) {
+    dout(10) << __func__ << ": quorum not formed" << dendl;
```

Comment:
i think this should just be dropped, actually.. the monmap doesn't know anything about quorum, and i can't tell what the intent was here